samples/charts/data-chart/polar-series/Pages/DataChartPolarSeries.razor

```diff
@@ -0,0 +1,244 @@
+@using IgniteUI.Blazor.Controls
+@inject IIgniteUIBlazor IgniteUIBlazor
+
+<div class="igContainer">
+
+    <div class="igOptions-horizontal">
+        <span class="igOptions-label">Type of Polar Series: </span>
+        <select value=@SeriesType
+                @onchange="OnSeriesTypeChanged">
+            <option>Area</option>
+            <option>Spline Area</option>
+            <option>Spline</option>
+            <option>Line</option>
+            <option>Scatter</option>
+        </select>
+
+        <span class="igLegend-title">Legend: </span>
+        <div class="igLegend">
+            <Legend @ref="LegendRef" Orientation="LegendOrientation.Horizontal" />
+        </div>
+    </div>
+
+    <div class="igComponent">
+        @if (DataSource != null)
+        {
```

Comment:
there should be 5 @if (ChartType == "PolarAreaSeries" ) conditions each with one type of series (see StackedSeries sample):
PolarAreaSeries, PolarScatterSeries, PolarLineSeries, PolarSplineSeries, PolarSplineAreaSeries,